lib/rubocop/cop/util.rb

```diff
@@ -152,6 +152,22 @@ def range_with_surrounding_space(range, side = :both, with_newline = true)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
+      def range_by_whole_lines(range, include_final_newline = false)
```

Comment:
> I'd make the second arg a keyword argument.

Will do. 👍 

> I also dislike the name of the method a bit, as I find it somewhat confusing. Why `range_by_whole_lines`?

Yeah I had trouble coming up with a name I liked too, and I'm also not too hot on this one. I called it `range_by_lines` at first but thought that seemed too vague. I also considered `range_with_whole_lines` but thought it sounded weird.

Do either of those sound better to you?